packages/markdown/remark/src/shiki.ts

```diff
@@ -43,18 +48,24 @@ export async function createShikiHighlighter({
 	theme = theme === 'css-variables' ? cssVariablesTheme() : theme;
 
 	const highlighter = await getHighlighter({
-		langs: langs.length ? langs : Object.keys(bundledLanguages),
+		langs: ['plaintext', ...langs],
 		themes: Object.values(themes).length ? Object.values(themes) : [theme],
 	});
 
-	const loadedLanguages = highlighter.getLoadedLanguages();
-
 	return {
-		highlight(code, lang = 'plaintext', options) {
+		async highlight(code, lang = 'plaintext', options) {
```

Comment:
I think this line in `Code.astro` will also need an await:

https://github.com/withastro/astro/blob/b3028caecf1a96310e2ee7a65075a81c231d87be/packages/astro/components/Code.astro#L90-L93

Not sure why the tests aren't failing for this.